docs/source/conf.py

```diff
@@ -69,7 +69,7 @@
 # so a file named "default.css" will overwrite the builtin "default.css".
 # html_static_path = ['_static']
 
-html_logo = os.path.join(root_dir, 'sunshine.png')
+html_logo = os.path.join(root_dir, 'sunshine.svg')
```

Comment:
I will need to double check the all the docs properly build using an SVG instead of PNG.

Reply:
Also, don't need to duplicate the svg file in the root of the directory. The png that's there is used for other things outside of the web UI.

Reply:
https://lizardbyte--980.org.readthedocs.build/projects/sunshine/en/980/#